### ISSUE TITLE
Charge status indicator for BLE

### ISSUE THREAD

**Issue by WebmasterTD** [mooltipass/moolticute#635]
### Missing feature
Would be helpful if in moolticute would be a BLE charge status indicator or just show the battery level.

### Justification
The device only has a 4 bar indicator, not very precise. If I want to check the battery level I have to open the Windows settings

**Comment by limpkin** [comment]
in theory the animation "starts" from the current battery state no? :)

**Comment by WebmasterTD** [comment]
Yes on the device the animation starts from current battery level, but indeed I'm thinking about moolticute on the PC.

**Comment by limpkin** [comment]
that's a nice idea
